src/index.js

```diff
@@ -0,0 +1,91 @@
+// import _ from "lodash";
+// import printMe from "./print.js";
+import './style.css';
+
+// function component() {
```

Comment:
- [ ] kindly please clean your comment code to be clear your code and readability 👍 .